dll/win32/shell32/stubs.cpp

```diff
@@ -136,6 +145,19 @@ WINAPI
 SHMultiFileProperties(IDataObject *pDataObject, DWORD dwFlags)
 {
     FIXME("SHMultiFileProperties() stub\n");
+
+    // Temporary workaround to display a property sheet when possible
```

Comment:
```suggestion
    // Temporary workaround to display a property sheet if possible
```
What is the condition that it is possible? And what is the condition that it is impossible?
Can you attach a screenshot or small video of the after-state? Thx.

Reply:
It is only possible if the dataobject contains a single item. It does nothing to help the case where there are multiple files.